redisenterprise/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,54 @@
+name: RedisEnterprise
+files:
+- name: redisenterprise.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/http
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/http
+    - template: instances/default
```

Comment:
It's better to put these at the end as they are more generic

Reply:
@hithwen I don't see any errors when syncing the config

```
$ ddev -x validate config --sync redisenterprise
Validating default configuration files...
$ echo $?
0
```

Reply:
@hithwen - added the other templates and removed tests that conflicted with this